.github/ISSUE_TEMPLATE/📲contribute-an-azd-template-.md

```diff
@@ -8,56 +9,32 @@ assignees: nitya, savannahostrowski
 ---
 
 ## New azd-template Submission
+Fill in the sections below to populate a new gallery card for your template. **All fields are mandatory unless explicitly identified as optional**
 
-Fill in the sections below to populate a new Gallery carry for your template. An example of a valid template object is given at the end for reference. **All fields are mandatory unless explicitly identified as optional**
+## 1. Description
+_1-2 sentence description of the architecture or solution that is defined by the template. If this is a requested template, please tag the GitHub issue containing the request._
 
-### 1. Title 
-_Short template name that will be used in card title on gallery_
+## 2. Author Information
+**Name:** _Name to credit on the gallery card_
 
-## 2. Description
-_1-2 sentence description of the architecture or solution that is defined by the template_
+**Link:** _To the author's GitHub profile (ideal) or other profile link_
 
-## 3. Author Profile
-_A link to the contributor's GitHub profile (ideal) or other profile link (for identification_.
+## 3. Template Repo
+_The GitHub repo containing the template being contributed-- must be a valid, complete template_
```

Comment:
What does "valid and complete" mean? How do we enforce that?

Reply:
good question! that's something leftover from before and I found myself wondering the same thing but figured it was something that had been discussed. I'll work on a different way to explain that templates will be reviewed by our team before they are approved

Reply:
makes sense - in the future (probably next month), we will spend some time building out better automated review tooling etc. for templates that can help handle some of this. Nothing to deal with yet but we can update again with those instructions when the time comes.